framework/src/outputs/Console.C

```diff
@@ -460,18 +460,26 @@ std::string
 Console::formatTime(const Real t) const
 {
   std::ostringstream oss;
-  if (_time_format == "plain" || _time_format == "second")
+  if (_time_format != "dtime")
   {
     if (_precision > 0)
       oss << std::setw(_precision) << std::setprecision(_precision) << std::setfill('0')
           << std::showpoint;
     if (_scientific_time)
       oss << std::scientific;
-    oss << t;
-    if (_time_format == "second")
-      oss << "s";
+
+    if (_time_format == "plain")
```

Comment:
I suggest using an enum class here and the `getEnum<...>()` call, like in [all other places where we frequently check enum values](https://github.com/search?q=repo%3Aidaholab%2Fmoose%20getEnum&type=code).